src/display.rs

```diff
@@ -233,6 +233,27 @@ where
         self.buffer[idx + 1] = low;
     }
 
+    /// Draw a line directly into the ssd1331 (bypassing the frame buffer) using it's
+    /// hardware acceleration.  This does no bounds checking.
+    pub fn draw_hw_line(&mut self, x1: u32, y1: u32, x2: u32, y2: u32, value: u16
+    ) -> Result<(), Error<CommE, PinE>> {
+        // TODO: should we try to clip the line to the display here?  that could be tricky.
```

Comment:
What behaviour does the display have in clipping cases? If it goes offscreen and doesn't do anything like wrapping around, it should be fine to not bother clipping. Presumably this also applies to `draw_hw_rect` too.

Reply:
Ok do we need to clip, or just abort, if any of it is off-screen?  I am unfamiliar with what is the standard for these drivers.  

Also, generally, do you think this is going to be helpful?  I am still on the fence, since mixing blit and hw accel causes noticeable flashing (at least on my softcore).

Reply:
I haven't done anything using both blitting and HW accel, so I can't comment on that. The `draw_hw_rect` method may be useful for the new embedded-graphics 0.7 features so would be worth keeping in IMO, although I can see complications arising when using both HW and the software buffer in this driver.